import/popups/PromptDialog.qml

```diff
@@ -18,7 +18,7 @@ Dialog {
     canAccept: input.text.length > 0
 
     property alias text: prompt.text
-    property alias value: prompt.value
+    property alias value: input.text
```

Comment:
If "value: input.text" is not updated, then that should imply that prompt value that is send back to the engine [1] is not right either.

[1]
https://github.com/sailfishos/sailfish-components-webview/blob/master/import/popups/PopupOpener.qml#L189

Reply:
That PromptDialog.value property is actually what's sent to the server, without this server always receives nothing from javascript prompt method.

Reply:
Yeap... and `CustomPromptPopup.qml` would need something like:
```
onAccepted: value = input.text
```
and removing that input.text binding.

Reply:
Maybe even:
```
Binding { target: popup; property: "value"; value: input.text }
```

Tried to rename "value" property to promptValue => works as well.